memcached-image/memcached-image.xsl

```diff
@@ -0,0 +1,118 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+<xsl:output method="xml" encoding="UTF-8" indent="yes"/>
+<xsl:template match="/">
+<xsl:comment> OBS-ExclusiveArch: aarch64 x86_64 </xsl:comment>
+<xsl:comment>
+  This file is generated,
+  don't manually modify this file,
+  please check README
+</xsl:comment>
+
+<image schemaversion="6.9" name="memcached-image" xmlns:suse_label_helper="com.suse.label_helper">
+
+  <description type="system">
+    <author>Thomas Bechtold</author>
+    <contact>tbechtold@suse.com</contact>
```

Comment:
maybe use a generic email address for contact?

Reply:
using cloud@suse.com for now and `SUSE OpenStack Cloud Team` as author. Does that work for you?